presto-docs/src/main/sphinx/release/release-320.rst

```diff
@@ -0,0 +1,67 @@
+===========
+Release 320
+===========
+
+General Changes
+---------------
+
+* Fix incorrect parameter binding order for prepared statement execution when
+  parameters appear inside a ``WITH`` clause. (:issue:`1191`)
+* Fix planning failure for certain queries involving a mix of outer and
+  cross joins. (:issue:`1589`)
+* Improve performance of queries containing complex predicates. (:issue:`1515`)
+* Avoid unnecessary evaluation of redundant filters. (:issue:`1516`)
+* Improve performance of certain window functions when using bounded window
+  frames (e.g., ``ROWS BETWEEN ... PRECEDING AND ... FOLLOWING``). (:issue:`464`)
+* Add :doc:`/connector/kinesis`. (:issue:`476`)
+* Add :func:`geometry_from_hadoop_shape`. (:issue:`1593`)
+* Add :func:`at_timezone`. (:issue:`1612`)
+* Add :func:`with_timezone`. (:issue:`1612`)
+
+JDBC Driver Changes
+-------------------
+
+* Only report warnings on ``Statement``, not ``ResultSet``, as warnings
+  are not associated with reads of the ``ResultSet``. (:issue:`1640`)
+
+CLI Changes
+-----------
+
+* Add multi-line editing and syntax highlighting. (:issue:`1380`)
+
+Hive Connector Changes
+----------------------
+
+* Add impersonation support for calls to the Hive metastore. (:issue:`43`)
```

Comment:
```
* Add impersonation support for calls to the Hive metastore. This can be enabled using the
  ``hive.metastore.thrift.impersonation.enabled`` configuration property. (:issue:`43`)